### DISCUSSION TITLE
Muchas gracias a los que han aceptado unirse a la comunidad!

### DISCUSSION THREAD

**Discussion by shirosweets** [FAMAF-resources/Welcome#2] [Anuncios]
Tienen alguna sugerencia o idea?

**Comment by IvanRenison** [comment]
Hay alguna convención para los nombres de archivo?

**Reply by AgustinMDominguez** [reply]
Es buena! 
Podemos armar una, al nivel de sugerencia.

Creo que debería priorizar que sea facil de encontrar algo particular y seguir el orden de la materia.
Creo que intentar mantener las cosas en el orden que se dan ayudaría bastante en eso:

Por ejemplo en redes vemos las cosas por capas, pero tienen un orden
0 - Introduccion
1 - Capa de Aplicacion
2 - Capa de Transporte 
...

entonces en el repo estaria bueno que estén tambien en ese orden

**Reply by shirosweets** [reply]
https://github.com/orgs/FAMAF-resources/discussions/9
